translations/ko.po

```diff
@@ -213,368 +229,368 @@ msgstr ""
 msgctxt "spell-trap-property"
 msgid "Normal Spell"
-msgstr ""
+msgstr "일반 마법"
 
 #: src\card.ts:52
 msgctxt "spell-trap-property"
 msgid "Continuous Spell"
-msgstr ""
+msgstr "지속 마법"
 
 #: src\card.ts:53
 msgctxt "spell-trap-property"
 msgid "Equip Spell"
-msgstr ""
+msgstr "장착 마법"
 
 #: src\card.ts:54
 msgctxt "spell-trap-property"
 msgid "Quick-Play Spell"
-msgstr ""
+msgstr "속공 마법"
 
 #: src\card.ts:55
 msgctxt "spell-trap-property"
 msgid "Field Spell"
-msgstr ""
+msgstr "필드 마법"
 
 #: src\card.ts:56
 msgctxt "spell-trap-property"
 msgid "Ritual Spell"
-msgstr ""
+msgstr "의식 마법"
 
 #: src\card.ts:57
 msgctxt "spell-trap-property"
 msgid "Normal Trap"
-msgstr ""
+msgstr "일반 함정"
 
 #: src\card.ts:58
 msgctxt "spell-trap-property"
 msgid "Continuous Trap"
-msgstr ""
+msgstr "지속 함정"
 
 #: src\card.ts:59
 msgctxt "spell-trap-property"
 msgid "Counter Trap"
-msgstr ""
+msgstr "카운터 함정"
 
 #: src\card.ts:95
 msgctxt "attribute"
 msgid "EARTH"
-msgstr ""
+msgstr "땅"
 
 #: src\card.ts:96
 msgctxt "attribute"
 msgid "WATER"
-msgstr ""
+msgstr "물"
 
 #: src\card.ts:97
 msgctxt "attribute"
 msgid "FIRE"
-msgstr ""
+msgstr "화염"
 
 #: src\card.ts:98
 msgctxt "attribute"
 msgid "WIND"
-msgstr ""
+msgstr "바람"
 
 #: src\card.ts:99
 msgctxt "attribute"
 msgid "LIGHT"
-msgstr ""
+msgstr "빛"
 
 #: src\card.ts:100
 msgctxt "attribute"
 msgid "DARK"
-msgstr ""
+msgstr "어둠"
 
 #: src\card.ts:101
 msgctxt "attribute"
 msgid "DIVINE"
-msgstr ""
+msgstr "신"
 
 #: src\card.ts:298
 #: src\card.ts:309
 msgctxt "card-embed"
 msgid "Card Text"
-msgstr ""
+msgstr "카드 텍스트"
 
 #: src\card.ts:303
 msgctxt "card-embed"
 msgid "Pendulum Effect"
-msgstr ""
+msgstr "펜듈럼 효과"
 
 #: src\card.ts:324
 msgctxt "card-embed"
 msgid "Card Effect"
-msgstr ""
+msgstr "카드 효과"
 
 #: src\locale.ts:46
 msgctxt "command-option"
 msgid "result-language"
-msgstr ""
+msgstr "결과언어"
 
 #: src\locale.ts:66
 msgctxt "command-option"
 msgid "input-language"
-msgstr ""
+msgstr "입력언어"
 
 #: src\commands\art.ts:38
 msgctxt "command-option"
 msgid "input"
-msgstr ""
+msgstr "입력"
 
 #: src\commands\ygoprodeck.ts:37
 msgctxt "command-option"
 msgid "term"
-msgstr ""
+msgstr "용어"
 
 #: src\commands\yugipedia.ts:40
 msgctxt "command-option"
 msgid "page"
-msgstr ""
+msgstr "페이지"
 
 #: src\locale.ts:49
 msgctxt "command-option-description"
 msgid "The output language for the card embed, overriding other settings."
-msgstr ""
+msgstr "다른 설정을 재정의하는 카드 임베드의 출력 언어입니다."
 
 #: src\locale.ts:69
 msgctxt "command-option-description"
 msgid "The language to search in, defaulting to the result language."
-msgstr ""
+msgstr "검색할 언어로, 기본적으로 출력 언어로 설정됩니다."
 
 #: src\commands\art.ts:41
 msgctxt "command-option-description"
 msgid "The password, Konami ID, or name to search for a card."
-msgstr ""
+msgstr "카드를 검색할 카드 코드, 코나미 ID, 또는 이름입니다."
 
 #: src\commands\ygoprodeck.ts:40
 msgctxt "command-option-description"
 msgid "The name or password of the card you're looking for."
-msgstr ""
+msgstr "찾고 있는 카드의 이름 또는 카드 코드입니다."
```

Comment:
```suggestion
msgstr "찾으시는 카드의 이름 또는 패스워드입니다."
```

Reply:
I'll fix at all.

Reply:
I fixed that.

-p.m. 7:18 | South Korea
-request by Ice-Pendragon